addon/-private/system/relationships/has-many.js

```diff
@@ -122,8 +122,8 @@ import { DEBUG } from '@glimmer/env';
   ```
 
   In contrast to async relationship, accessing a sync relationship
-  will never trigger a request to fetch the resources,
-  and it will always return an array object with the existing local resources.
+  will always return a [DS.ManyArray](/api/data/classes/DS.ManyArray.html) instance
+  containing the existing local resources, or null.
```

Comment:
I don't think this returns `null` if no resources are available locally, but rather the array will be empty.

Similarly to `belongsTo`, I think we should say something about the fact that we error here if you access the relationship prior to the related resources being available (see my original feedback for suggested text).